docusaurus/docs-internals/editor-controls/data-controls/motion.md

```diff
@@ -190,3 +254,178 @@ An array of animation names that should be disabled. In this case, you cannot se
   }
 }
 ```
+
+#### Default value examples
+
+In this example, the `motion` control that has the values for `"rotate"` and `"speed"` enabled by default and `"rotate"` is active.
+`"rotate"` has `"speed"` = `13` and `"direction"` = `"right"`, `"scale"` has `"speed"` = `7`.
+
+```js
+{
+  id: "motion", 
+  type: "motion",
+  default: {
+    active: "rotate",
+    verticalEnabled: false,
+    verticalSpeed: 0.5,
+    verticalSpeedSuffix: null,
+    verticalDirection: "up",
+    verticalViewportType: "viewport",
+    verticalViewportTop: 0,
+    verticalViewportBottom: 100,
+    horizontalEnabled: false,
+    horizontalSpeed: 0.5,
+    horizontalSpeedSuffix: null,
+    horizontalDirection: "left",
+    horizontalViewportType: "viewport",
+    horizontalViewportBottom: 100,
+    horizontalViewportTop: 0,
+    rotateEnabled: false,
+    rotateSpeed: 0.5,
+    rotateSpeedSuffix: null,
+    rotateDirection: "left",
+    rotateX: "center",
+    rotateY: "center",
+    rotateViewportType: "viewport",
+    rotateViewportBottom: 100,
+    rotateViewportTop: 0,
+    transparencyEnabled: false,
+    transparencyLevel: 0.5,
+    transparencyLevelSuffix: null,
+    transparencyDirection: "in",
+    transparencyViewportType: "viewport",
+    transparencyViewportBottom: 100,
+    transparencyViewportTop: 0,
+    scaleEnabled: false,
+    scaleSpeed: 10,
+    scaleSpeedSuffix: null,
+    scaleDirection: "up",
+    scaleX: "center",
+    scaleY: "center",
+    scaleViewportType: "viewport",
+    scaleViewportBottom: 100,
+    scaleViewportTop: 0,
+    blurEnabled: false,
+    blurLevel: 0.5,
+    blurLevelSuffix: null,
+    blurDirection: "in",
+    blurViewportType: "viewport",
+    blurViewportBottom: 100,
+    blurViewportTop: 0,
+    mouseTrackEnabled: false,
+    mouseTrackDirection: "direct",
+    mouseTrackSpeed: 0.25,
+    mouseTrackSpeedSuffix: null,
+    mouseTiltEnabled: false,
+    mouseTiltDirection: "direct",
+    mouseTiltSpeed: 0.25,
+    mouseTiltSpeedSuffix: null
+  }
+}
+```
+
+#### CSS examples
+
+Change the `.brz-text` element color with CSS using a `motion` control value.
+If `rotate` speed value >= 0.7, then we will colour `.brz-text` element to red, otherwise it will be blue.
+Also, if `transparency` level >= 0.5, then we will set `.brz-box` element to display `"flex"`, otherwise it will be displayed `"none"`;
+
+```js
+{
+  id: "motion",
+  type: "motion",
+  style: ({ value }) => {
+    const brzTextColor = value?.rotate?.speed >= 0.7 ? "red" : "blue";
```

Comment:
use more specific example, because speed with color did not relate

write an example with `transform` and `horizontal` or `vertical` values of your control